source/develop/release-management/features/network/provider-physical-network.md

```diff
@@ -79,6 +79,20 @@ Engine also covers the validation:
 - Physical network must be on the same Data Center as the external network.
 - The custom values (physical network and VLAN ID) must not be specified a when provider physical network is set.
 
+To import a network with physical network access from an external network
+provider into engine's data base, this physical network has to be mapped to the
+corresponding logical network in engine.
+The [external network provider][1] describes the physical network with the
+three attributes `provider:physical_network`, `provider:network_type` and
+`provider:segmentation_id`.
+The corresponding logical network is detected in the context of a data center
+by checking if there is a logical network in the data center with the
+[VDSM name][2] like the `provider:physical_network` attribute, which is from
+the same VLAN type like the `provider:network_type` attribute and has the same
```

Comment:
It is not necessarily a VLAN type. What about provider:network_type=flat in case of non-VLAN networks.

Reply:
Done.